cypress-tests/cypress/e2e/admin/pageBuilder/templates/pageTemplatesCrud.cy.ts

```diff
@@ -0,0 +1,61 @@
+context("Page Builder - Blocks", () => {
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+    });
+
+    it("Should be able to create a template and then edit and delete it", () => {
+        cy.visit("/page-builder/page-templates");
+        //Creates a template using the UI.
+        cy.findAllByTestId("pb-templates-list-new-template-btn").eq(0).click();
+        cy.findByRole("textbox", { name: "Title" }).type("testingfunctionality");
+        cy.findByRole("textbox", { name: "Slug" }).type("testingfunctionality");
+        cy.findByRole("textbox", { name: "Description" }).type("testingfunctionality");
+        cy.findByRole("button", { name: "Create" }).click();
+        cy.findByRole("button", { name: "Save Changes" }).should("exist").click();
+
+        cy.contains("testingfunctionality").should("exist");
+
+        //Edits the template name using the UI.
```

Comment:
```suggestion
        // Edits the template name using the UI.
```

Please apply the same for the rest of the comments.